SoftLayer/managers/block.py

```diff
@@ -616,3 +616,20 @@ def create_or_update_lun_id(self, volume_id, lun_id):
         """
         return self.client.call('Network_Storage', 'createOrUpdateLunId',
                                 lun_id, id=volume_id)
+
+    def refresh_dep_dupe(self, volume_id, snapshot_id):
```

Comment:
Since these 2 commands are literally the same for file/block, just put them in the storage_utils.py file so we don't have as much duplicated code in these 2 managers than we already do.

Reply:
fixed, storage.py manager though, not utils